bitbots_body_behavior/src/bitbots_body_behavior/decisions/game_state_decider.py

```diff
@@ -26,7 +26,17 @@ def perform(self, reevaluate=False):
             return "NOT_ALLOWED_TO_MOVE"
 
         game_state_number = self.blackboard.gamestate.get_gamestate()
-        return self.game_states[game_state_number]
+        #todo this is a temporary hack to make GUI work
+        if game_state_number == 0:
```

Comment:
I would prefer it if you used the definitions from the message instead of the magic numbers